source/expressions.tex

```diff
@@ -754,7 +754,7 @@
 \item
 if the resulting $\cv{}^3_i$ is different from $\cv{}^1_i$ or $\cv{}^2_i$,
 or the resulting $P^3_i$ is different from $P^1_i$ or $P^2_i$,
-then \tcode{const} is added to every $\cv{}^3_k$ for $0 < k < i$.
+then \tcode{const} is added to every $\cv{}^3_k$ for $0 < k < i$;
```

Comment:
I think:

* This is connecting the bullets to the "where", so we don't need to use the same character as for the other bullets here. I'd expect a comma rather than a semicolon before a "where".
* We're missing an "and" at the end of the second bullet.
* It would be fine to use commas rather than semicolons for the first two bullets. The bullets are sufficient to indicate the structure here.

Reply:
fixed